src/main/resources/org/ysb33r/gradletest/GradleTestTemplate.groovy.template

```diff
@@ -123,6 +123,17 @@ class ${TESTNAME.replaceAll(/\W/,'_')}CompatibilitySpec extends Specification {
         }
     }
 
+    def getDistributionFilename(String version){
```

Comment:
New code in the template file that does the actual version replace for the filename pattern and returns the final file name